### DISCUSSION TITLE
支持禁用部分设备的在线提醒

### DISCUSSION THREAD

**Discussion by TigerBeanst** [XiaoMi/ha_xiaomi_home#200] [Ideas]
![image](https://github.com/user-attachments/assets/e7cfe6af-e2ee-490f-87a5-288909c212e4)

因为有些设备并不是长期在线的，希望可以在检测在线设备的时候忽略这些设备

这个需求不是屏蔽设备，只是屏蔽设备的在线检测

**Comment by pcr** [comment]
赞同，我的小米电视不支持长待机，关机后每次都在HA里提醒设备掉线。

**Reply by ooodwbooo** [reply]
我也是，后来我把小米集成的离线提醒关了

**Comment by refined-fish** [comment]
可以一次性全关了，但好像还不能关闭特定设备，不过我为了随时可用好多东西都没断电（浪费电ing😂）
